plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/ExBudgetMode.hs

```diff
@@ -136,7 +145,7 @@ restricting (ExRestrictingBudget (ExBudget cpuInit memInit)) = ExBudgetMode $ do
                 throwingWithCauseExc @(CekEvaluationException uni fun) _EvaluationError
```

Comment:
That's because the CPU budget you set, although it looks like a large number, is less than the startup cost. So it fails immediately after spending that, and that step uses no memory. So if you set the CPU budget a big higher but still less than what it needs, you will see it does spend memory.

Reply:
```
~/projects/iohk/plutus slippage ⇣1⇡47 ?1 ❯ cabal run plc evaluate -- --if flat -i ../plutus-scripts/crowdfunding-success-1.flat -R 100000000:10000                                                                                                                                                                       ▼
Up to date
CPU budget:    -6000000
Memory budget: 2000
An error has occurred:  User error:
The budget was overspent. Final negative state: ({ cpu: -6000000 | mem: 2000 })
```

Reply:
> less than the startup cost.

Of course.  I'd forgotten all about that.  I'm hoping that might go away soon, or at least get a lot smaller. We've lost that expensive annotation traversal for one thing.  "Startup" is a bit of a misnomer anyway: it's more "unexplained overhead".